std/typecons.d

```diff
@@ -4256,6 +4256,49 @@ if (!is(T == class) && !(is(T == interface)))
             _store._count = 1;
         }
 
+        private void move(ref T source)
+        {
+            import core.memory : GC;
+            import core.stdc.stdlib : malloc;
+            import core.stdc.string : memcpy;
+            import std.exception : enforce;
+
+            _store = cast(Impl*) enforce(malloc(Impl.sizeof));
```

Comment:
You can't use `enforce` for `malloc`, use core.exception.onOutOfMemoryError.

See also #3031.